app/controllers/mixins/middleware_common_mixin.rb

```diff
@@ -105,11 +105,22 @@ def run_operation_on_record(operation_info, item_record)
     end
   end
 
+  def skip_operation?(item_record, operation_info)
+    provider_product_name = 'Hawkular'.freeze
+    if operation_info.fetch(:skip)
+      if item_record.has_attribute?('product')
+        item_record.product == provider_product_name
+      elsif item_record.respond_to?('middleware_server')
+        item_record.middleware_server.product == provider_product_name
+      end
+    end
```

Comment:
This can be shortened to:

```ruby
if operation_info.fetch(:skip)
  item_record.try(:product) == name || item_record.try(:middleware_server).try(:product) == name
end
```